SlingingPlatformGame/Assets/Scripts/PlayerMovement.cs

```diff
@@ -112,7 +112,7 @@ private void OnCollisionExit2D(Collision2D other)
         Debug.Log(other.gameObject);
         if (other.gameObject.CompareTag("Ground"))
         {
-            isJumping = true;
+            // isJumping = true;
```

Comment:
This if condition

Reply:
I want to keep it because I am not sure whether the change solves the issue or not. I think it should but can't say.